Tests/ValidationTests.swift

```diff
@@ -360,10 +358,9 @@ class MultipleValidationTestCase: BaseTestCase {
         // Then
         XCTAssertNotNil(error)
 
-        if let error = error {
-            XCTAssertEqual(error.domain, ErrorDomain)
-            XCTAssertEqual(error.code, ErrorCode.statusCodeValidationFailed.rawValue)
-            XCTAssertEqual(error.userInfo[ErrorUserInfoKeys.StatusCode] as? Int, 200)
+        if let error = error as? AFError, let statusCode = error.responseCode {
+            XCTAssertTrue(error.isUnacceptableStatusCode)
+            XCTAssertEqual(statusCode, 200)
         } else {
             XCTFail("error should not be nil")
         }
```

Comment:
That won't work, as `XCTAssertEqual` doesn't work on optional and non-optional values.